src/main/java/net/comments/objects/CommentErrors.java

```diff
@@ -0,0 +1,72 @@
+package net.comments.objects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+
+public class CommentErrors {
+
+    private final WebDriver driver;
+
+    public CommentErrors(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    //The Comment Text field should contain alphanumeric characters only
+    public boolean hasTextValidationError() {
```

Comment:
I'm not sure if you need `has...` methods at all. Just run `getErrorText()` in case you need to check the existence of the error.